docs/ha-mode.md

```diff
@@ -109,8 +97,8 @@ Access endpoints are evaluated automagically, as the following:
 
 | Endpoint type                | kube-master   | non-master          |
 |------------------------------|---------------|---------------------|
-| Local LB (overrides ext)     | http://lc:p   | http://lc:p         |
-| External LB, no internal     | https://lb:lp | https://lb:lp       |
+| Local LB                     | http://lc:p   | http://lc:sp        |
+| External LB, no internal     | http://lc:p   | https://lb:lp       |
 | No ext/int LB (default)      | http://lc:p   | https://m[0].aip:sp |
```

Comment:
I'm failing to find the code reflecting the change for kube-master's https://lb:lp to http://lc:p . It should be put to https://github.com/kubespray/kargo/pull/528/files#diff-a902856212453b3541060b8a6265c074R17 perhaps